recipes/funcsigs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
```

Comment:
This is a bit weird. However, if we don't do this, we will run into this issue ( https://github.com/conda/conda-build/issues/835 ).